models/junos/security_policy/junos_security_policies.yml

```diff
@@ -0,0 +1,514 @@
+---
+GENERATOR_VERSION: '1.0'
+ANSIBLE_METADATA: |
+  {
+     'metadata_version': '1.1',
+     'status': ['preview'],
+     'supported_by': 'network'
+  }
+NETWORK_OS: junos
+RESOURCE: l3_interfaces
```

Comment:
module is security policy right?

Reply:
yeah i noticed this yesterday night, already fixed